2020-12-24 17:25:24,298 INFO    MainThread:10132 [wandb_setup.py:_flush():70] setting env: {}
2020-12-24 17:25:24,298 INFO    MainThread:10132 [wandb_setup.py:_flush():70] multiprocessing start_methods=spawn
2020-12-24 17:25:24,298 INFO    MainThread:10132 [wandb_setup.py:_flush():70] setting login settings: {}
2020-12-24 17:25:24,298 INFO    MainThread:10132 [wandb_init.py:_log_setup():303] Logging user logs to C:\Users\deepu\Desktop\MyFolders\Projects\PetProjects\ActivityRecognition\src\wandb\run-20201224_172524-myuwswxb\logs\debug.log
2020-12-24 17:25:24,298 INFO    MainThread:10132 [wandb_init.py:_log_setup():304] Logging internal logs to C:\Users\deepu\Desktop\MyFolders\Projects\PetProjects\ActivityRecognition\src\wandb\run-20201224_172524-myuwswxb\logs\debug-internal.log
2020-12-24 17:25:26,565 INFO    MainThread:10132 [wandb_run.py:_console_start():1237] atexit reg
2020-12-24 17:25:26,567 INFO    MainThread:10132 [wandb_run.py:_redirect():1104] redirect: SettingsConsole.WRAP
2020-12-24 17:25:26,567 INFO    MainThread:10132 [wandb_run.py:_redirect():1135] Wrapping output streams.
2020-12-24 17:25:26,567 INFO    MainThread:10132 [wandb_run.py:_redirect():1151] Redirects installed.
2020-12-24 17:25:26,972 INFO    MainThread:10132 [wandb_run.py:_atexit_cleanup():1207] got exitcode: 1
2020-12-24 17:25:26,972 INFO    MainThread:10132 [wandb_run.py:_restore():1179] restore
2020-12-24 17:25:26,984 INFO    MainThread:10132 [wandb_run.py:_wait_for_finish():1324] got exit ret: uuid: "8340828006b94e2ebd13015154a80fba"
response {
  poll_exit_response {
    file_counts {
      wandb_count: 1
    }
    pusher_stats {
      total_bytes: 794
    }
  }
}

2020-12-24 17:25:28,987 INFO    MainThread:10132 [wandb_run.py:_wait_for_finish():1324] got exit ret: uuid: "d9101459576a44579f9b876d1b3a381f"
response {
  poll_exit_response {
    done: true
    exit_result {
    }
    file_counts {
      wandb_count: 5
    }
    pusher_stats {
      uploaded_bytes: 6613
      total_bytes: 6613
    }
  }
}

2020-12-24 17:25:30,127 INFO    MainThread:10132 [wandb_run.py:_show_files():1536] logging synced files
2020-12-24 17:25:30,131 INFO    MainThread:10132 [internal.py:handle_exit():138] Internal process exited
